packages/cozy-mespapiers-lib/src/components/Papers/helpers.js

```diff
@@ -127,21 +127,21 @@ export const buildFilesByContacts = ({
   t
 }) => {
   const result = []
-  const isAloneContactOrKonnector =
-    [...contacts, ...konnectors].length <= 1 && !hasMultipleAccounts
-
   const {
     itemsFound: filesCreatedByKonnectors,
     remainingItems: filesNotCreatedByKonnectors
   } = filterWithRemaining(files, isFromKonnector)
 
-  if (filesCreatedByKonnectors.length > 0) {
-    const filesByKonnectors = groupBy(
-      filesCreatedByKonnectors,
-      file =>
-        `${file.cozyMetadata.createdByApp}-${file.cozyMetadata.sourceAccountIdentifier}`
-    )
+  const filesByKonnectors = groupBy(
+    filesCreatedByKonnectors,
+    file =>
+      `${file.cozyMetadata.createdByApp}-${file.cozyMetadata.sourceAccountIdentifier}`
+  )
+  const isAloneContactOrKonnector =
+    contacts.length + Object.keys(filesByKonnectors).length <= 1 && // Check if there is only one contact or one konnector/account in all files created by konnectors (may have konnectors that are no longer connected/functional)
```

Comment:
nit: Would have added () around the addtion

Reply:
Eslint disagrees 😁